mock-abis/src/main/java/io/mosip/proxy/abis/service/impl/ProxyAbisInsertServiceImpl.java

```diff
@@ -390,24 +365,35 @@ public IdentifyDelayResponse findDuplication(IdentityRequest ir) {
 			logger.info("find duplicate property set to " + proxyAbisConfigService.getDuplicate());
 			logger.info("force duplicate property set to " + proxyAbisConfigService.isForceDuplicate());
 			if (ir.getGallery() != null && ir.getGallery().getReferenceIds().size() > 0
-					&& ir.getGallery().getReferenceIds().get(0).getReferenceId() != null 
+					&& ir.getGallery().getReferenceIds().get(0).getReferenceId() != null
 					&& !ir.getGallery().getReferenceIds().get(0).getReferenceId().isEmpty()) {
 				List<String> referenceIds = new ArrayList();
 				ir.getGallery().getReferenceIds().stream().forEach(ref -> referenceIds.add(ref.getReferenceId()));
 
 				logger.info("checking for duplication of reference Id against" + referenceIds.toString());
+
+				int galleryRefIdCountInDB = proxyAbisBioDataRepository
+						.fetchCountForReferenceIdPresentInGallery(referenceIds);
+				if (galleryRefIdCountInDB != referenceIds.size()) {
+					logger.info("checking for reference Id Present in DB ==== " + galleryRefIdCountInDB);
```

Comment:
Don't use logger with +

Reply:
logger.info(String.format("checking for reference Id Present in DB[%d], Gallery reference Id list size[%d] ", galleryRefIdCountInDB, referenceIds.size()));